substrate/frame/revive/src/exec.rs

```diff
@@ -925,6 +926,7 @@ where
 		deposit_limit: BalanceOf<T>,
 		read_only: bool,
 		origin_is_caller: bool,
+		is_root: bool,
```

Comment:
No need to add this parameter.  `origin_is_caller` is equivalent.

Reply:
Oversight on my part - thanks for pointing it out.